src/main/scala/com/outr/lucene4s/query/MoreLikeThisConfig.scala

```diff
@@ -0,0 +1,16 @@
+package com.outr.lucene4s.query
+
+import org.apache.lucene.queries.mlt.MoreLikeThis
+
+case class MoreLikeThisConfig(minTermFreq: Int = MoreLikeThis.DEFAULT_MIN_TERM_FREQ,
```

Comment:
What about, instead of a `MoreLikeThisConfig`, `MoreLikeThisSearchTerm` were to have all of those values included? The `mlt` method could define default values so you could simplify usage to something like:

`mlt(value, boost = true, minWordLen = 5)`

Reply:
Thanks, I will change that.